### DISCUSSION TITLE
UTF-16 export to CSV bug

### DISCUSSION THREAD

**Discussion by p0c07o** [dbeaver/dbeaver#14379] [Q&A] [answered]
When exporting a result set to CSV using UTF-16 encoding, the first cell (i.e. A1) of the output document has additional content:

![image](https://user-images.githubusercontent.com/5910496/135598005-f8349dce-1c0d-4706-b84b-65dd38b18dc1.png)

When doing the same export using UTF-8 for example, the A1 cell is always correct. For example:

![image](https://user-images.githubusercontent.com/5910496/135598114-83354bb9-f1c5-4ef8-8a5b-1749876cae80.png)

**Comment by LonwoLonwo** [comment]
Hello @p0c07o 
What DBeaver version do you use?

**Comment by ShadelessFox** [comment, marked as answer]
Hello @p0c07o,

Perhaps you have `Insert BOM` option enabled?

Also please note that encodings like `UTF-16LE` and `UTF-16BE` (instead of `UTF-16`) may cause such results.

![image](https://user-images.githubusercontent.com/35821147/135603007-f2278c76-ea33-463a-8f54-d1524655290b.png)
